packages/expo-media-library/ios/EXMediaLibrary/EXMediaLibrary.m

```diff
@@ -128,6 +129,24 @@ - (id)requesterClass:(BOOL)writeOnly
                                                                 reject:reject];
 }
 
+UM_EXPORT_METHOD_AS(presentLimitedLibraryPickerAsync,
```

Comment:
I know this name is taken from `PHPhotoLibrary`'s method that you call, but it really sucks — it's kinda confusing in what it does. What about `presentPermissionsPickerAsync`? Or even making it a part of `requestPermissionsAsync` if the current permission is set to limited?